app/models/agenda-item.ts

```diff
@@ -8,9 +14,15 @@ export default class AgendaItemModel extends Model {
   @attr('string') declare alternateLink: string;
   @attr('boolean') declare plannedPublic: boolean;
 
-  @belongsTo('session', { async: true, inverse: 'agendaItems' })
-  declare session?: SessionModel;
+  @hasMany('session', { async: true, inverse: 'agendaItems' })
+  declare sessions?: AsyncHasMany<SessionModel>;
 
   @belongsTo('agenda-item-handling', { async: true, inverse: null })
-  declare handledBy?: AgendaItemHandlingModel;
+  declare handledBy?: AsyncBelongsTo<AgendaItemHandlingModel>;
+
+  get session() {
```

Comment:
**Question:** This getter is meant to then return the first session it can find for the agenda-item, right?